api-reference/v2/api-reference-guide.md

```diff
@@ -0,0 +1,213 @@
+
+### "Versioning" section
+
+This section is optional. If applicable, use this section to describe the versioning, compatibility, and lifecycle policies of your API set. If migration guides are available, provide the links here.
+
+
+### "Pagination" section
+
+This section is optional. If applicable, use this section to describe the options and default values for dividing long API responses into pages.
+
+
+### "Rate limiting and throttling" section
+
+This section is optional. If applicable, use this section to describe the global rate limiting settings and quota of your APIs. If different quota applies to different endpoints, also add this section in the reference for endpoints.
+
+
+### "Status code" section
+
+**Tips**:
+
+- For REST APIs, HTTP status codes are used. If you are writing references for other protocols, adjust the table columns.
+- In the `Description` column, describe the causes of that error, and provide related information or links about how to deal with the error.
+
+### Additional sections
+
+If other specifications are applicable to the whole API set, such as data conventions, request retry logic, logging, monitoring, and so on, define your own sections at the end of this document.
+
+
+## About the API resource reference section
+
+The template assumes that the API endpoints are grouped by the resource type that they are interacting with.
+
+**Title**
+
+Substitute the document title `{Resource Name}` with the actual name.
+
+**Short description**
+
+Provide a one-line definition of the resource in the opening section and explain how and when your users should use it.
+
+### "Data model" section
+
+Use this section to provide specifications of the resource entity in the table:
+
+- **Attribute**: the field name or property name defined by the resource
+- **Type**: data type of the value
+- **Required/Optional**: whether the attribute is a required field or not. Use a distinctive font to highlight the required attributes. In this template, capitalized letters are used.
+- **Description**: additional information such as whether the attribute has default values, validation restrictions, and whether it is a non-editable field that cannot be updated with POST requests.
+
+If a cell is empty, fill in “N/A”.
+
+### "Example" section
+
+Provide a concrete example of the data entity with valid values. Fill in as many optional attributes as possible.
+
+Use fenced code blocks (a pair of three backticks `` ``` ``) to make your code distinctive from other text blocks.
+
+Many Markdown processors also support syntax highlighting, which adds color to keywords. Indicate the language mode of your example after the opening backticks to take advantage of this feature. For example: `` ```json ``. The displayed color schema depends on your processor and the rendering configurations.
+
+### "Endpoints" section
+
+List the endpoints that can interact with this resource type in the table:
+
+- Capitalize the method names.
```

Comment:
Here, the "method name" refers to the  "GET" "POST" keywords, which are used in the capitalized format by convention, but the endpoint name may use sentence heading. Added an example to avoid confusion.